app/views/curation_concerns/base/_representative_media.html.erb

```diff
@@ -1,7 +1,5 @@
 <% if work.representative.present? && (gf = ::GenericFile.load_instance_from_solr(work.representative)) %>
   <%= render partial: 'curation_concerns/generic_files/media_display', locals: {generic_file: gf, dom_class_name: 'representative_image'} %>
-<% elsif work.representative.present? %>
```

Comment:
Why are you dumping this part of the condition rather than tacking `main_app.` onto `download_path`?